src/components/SingleArticle.jsx

```diff
@@ -123,6 +155,18 @@ const SingleArticle = ({ loggedIn }) => {
 											comments.map((comment) => {
 												return <Comment comment={comment} />;
 											})}
+										<form className='flex' onSubmit={(e) => onSubmitHandler(e)}>
+											<input
```

Comment:
would consider a text area instead of input so it can be multi-line (seems better for comments)